Makefile

```diff
@@ -6,6 +6,8 @@ lint:
 
 test: generate lint
 	go test -race -cover ./...
+	go build ./cmd/tengo
+	./tengo ./cmd/tengo/testdata/test.tengo
```

Comment:
I think these test .tengo files can be in `testdata` directory at project root.

Reply:
Why not